src/renderers/native/ReactNativeFiber.js

```diff
@@ -459,12 +87,31 @@ const ReactNative = {
   },
 
   unstable_batchedUpdates: ReactGenericBatching.batchedUpdates,
+
+  __SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED: {
+    // Used for Flow types
+    SyntheticEvent: require('SyntheticEvent'),
+
+    // Used as a mixin in many createClass-based components
+    NativeMethodsMixin: require('NativeMethodsMixin'),
+
+    // Used by react-native-github/Libraries/ components
+    PooledClass: require('PooledClass'), // Components/Touchable
```

Comment:
Can we copy and paste `PooledClass` into RN repo? It is relatively small and completely isolated.
I'd like to avoid treating it as part of React's contract, and be able to safely change or remove it.